ArcGISRuntimeSDKQt_CppSamples/Analysis/TraceUtilityNetwork/TraceUtilityNetwork.cpp

```diff
@@ -220,13 +268,20 @@ void FindFeaturesUtilityNetwork::onIdentifyLayersCompleted(QUuid, const QList<Id
     return;
   }
 
+  // Get domain network
+  const UtilityDomainNetwork * domainNetwork = m_utilityNetwork->definition()->domainNetwork("ElectricDistribution");
+  m_mediumVoltageTier = domainNetwork->tier(QString("Medium Voltage Radial"));
+
   const IdentifyLayerResult* result = results[0];
   m_feature = static_cast<ArcGISFeature*>(result->geoElements()[0]);
   UtilityElement* element = nullptr;
   const UtilityNetworkSource* networkSource = m_utilityNetwork->definition()->networkSource(m_feature->featureTable()->tableName());
 
   if (networkSource->sourceType() == UtilityNetworkSourceType::Junction)
   {
+    m_juncSelected = true;
+    emit juncSelectedChanged();
```

Comment:
Optional, but these don't seem so long that they need abbreviation. I think `junction` would help readability a bit, although this is not too bad like this.